### PR TITLE
Creates intial project folders and files

### PR DESCRIPTION
**Pull request by MuroriM**
Creates the files and folders

### CODE REVIEW

**Review by ash14-js — APPROVED** [review]

File: Fit/pom.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      <artifactId>javax.servlet-api</artifactId>
+      <version>4.0.1</version>
+      <scope>provided</scope>
+    </dependency>
+
+    <dependency>
+        <groupId>com.google.code.gson</groupId>
+        <artifactId>gson</artifactId>
+        <version>2.8.6</version>
+    </dependency>
```
Comment (hosungs) [inline Fit/pom.xml]: test review comment 1

**Review by matta628 — APPROVED** [review]
LGTM

**Review by hosungs — COMMENTED** [review]
Let's put this PR on hold until we have a good `.gitignore` so that we don't get many unnecessary files (e.g., binaries like `.class`) in our repo. I'll send a PR for that soon.

**Comment by hosungs** [comment]
As I mentioned at our meeting, any one of you can click the "Merge pull request" button as long as there are 2 approvals, but let's not do that this time, until we get the `.gitignore` file in correctly and there's no unnecessary files (e.g., `.class` binary files) in this PR. I left a message to @MuroriM for this, so let's wait Murori's follow-up. Thanks!

**Comment by hosungs** [comment]
Thanks @MuroriM for merging master and push your branch, but apparently that didn't remove the unnecessary binary files. I'll fix that myself, so no worries, and this time, I just merged/closed the PR. Next time, that should be all among yourselves. Thanks again and keep going!

**Pull request merged**